typescript/25-simple-cipher/simple-cipher.ts

```diff
@@ -0,0 +1,44 @@
+export class SimpleCipher {
+  public key: string;
+  private static alphabet = 'abcdefghijklmnopqrstuvwxyz';
+
+  constructor(key?: string) {
+    this.key = key || SimpleCipher.generateRandomKey();
+  }
+
+  static generateRandomKey(length = 100): string {
+    let result = '';
+    for (let i = 0; i < length; i++) {
+      result += this.alphabet.charAt(Math.floor(Math.random() * this.alphabet.length));
+    }
+    return result;
+  }
+
+  private shift(char: string, shift: number): string {
+    const charIndex = SimpleCipher.alphabet.indexOf(char);
+    const newIndex = (charIndex + shift + 26) % 26;
```

Comment:
26 is a magic number, better to get rid of it - think of how can you achieve that